plugins/browser-plugin-media/src/api.ts

```diff
@@ -396,7 +396,9 @@ export function trackMediaAdSkip(
   trackers: Array<string> = Object.keys(_trackers)
 ) {
   let { percentProgress } = args;
-  if (percentProgress !== undefined) { percentProgress = Math.floor(percentProgress); }
```

Comment:
Can you remove these styling changes from the PR?

Reply:
It does this on the Pre-commit CI now, so not really!